src/sensors/IMD.py

```diff
@@ -0,0 +1,142 @@
+from pin.pinout import Pinout
+from services.InputCapture import InputCapture
+from services.digital_in import DigitalInService
+from services.digital_out import DigitalOutService
+from shared_memory import SharedMemory
+from src.pin import memory
+from test_lib.input.aggregate import Multiple
+from test_lib.input.input import Input, InputFailedException
+from test_lib.condition.condition import Condition
+from test_lib.input.timed import After
+from enum import Enum
+
+
+class Case(Enum):
+    ShortCircuit = 0
+    Normal = 10
+    UnderVoltage = 20
+    EquipmentFault = 40
+    GroundFault = 50
```

Comment:
I don't understand the values of each enum variant, should be left to default if you are not using the value for anything

Reply:
There is no default value in python enums. If you want, I can change it to 1, 2, 3... values, or import auto function to assign a value automatically, which will do the same, assign 1, 2, 3...